content/en/docs/tasks/inject-data-application/podpreset.md

```diff
@@ -26,157 +23,289 @@ Get an overview of PodPresets at
 {{% capture steps %}}
 
 
-## Simple Pod Spec Example
+## Use Pod presets to inject environment variables and volumes
 
-This is a simple example to show how a Pod spec is modified by the Pod
-Preset.
+In this exercise, you create a Pod preset that has one environment variable and a volume mount.
+Here is the configuration file for the PodPreset:
 
 {{< codenew file="podpreset/preset.yaml" >}}
 
-Create the PodPreset:
+In the configuration file, you can see that the preset has an environment variable definition called `DB_PORT`
+and a volume mount definition called `cache-volume` which is mounted under `/cache`. The `selector` specifies that
```

Comment:
Optionally: use a glossary tooltip for “selector”: `{{< glossary_tooltip text="selector" term_id="selector" >}}`

Reply:
Tooltip has been added